### PR TITLE
PR#21572 continued - Added real-time kernel support (epic GRPA-1211)

### PR DESCRIPTION
**Pull request by ahardin-rh**
Continuation of https://github.com/openshift/openshift-docs/pull/21572

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@gpei Can you please provide QE review? Thank you!

**Comment by gpei** [comment]
@shellyyang1989 could you help on this Doc PR?

File: modules/installation-special-config-rtkernel.adoc [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+Linux real-time kernel when you first install the cluster. To do this,
+create a MachineConfig object and inject that object into the set of manifest
+files used by Ignition during cluster setup, as described in the following
+procedure.
+
+[NOTE]
+====
+This procedure is fully supported with bare metal installations using
+systems that are certified for Red Hat Enterprise Linux for Real Time 8.
+Real time support in {product-title} is also limited to specific subscriptions.
```
Comment (shellyyang1989) [inline modules/installation-special-config-rtkernel.adoc:25]: @ahardin-rh The feature also works on GCP and the procedure is also supported with GCP. So I suggest to add GCP in the NOTE

File: modules/installation-special-config-rtkernel.adoc [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+====
+
+.Prerequisites
+* For a bare metal installation of {product-title}, prepare masters and workers.
+* Use {product-title} version 4.4 or later.
```
Comment (shellyyang1989) [inline modules/installation-special-config-rtkernel.adoc:30]: Same suggestion as last comment covering GCP

File: modules/installation-special-config-rtkernel.adoc [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+.Procedure
+
```
Comment (shellyyang1989) [inline modules/installation-special-config-rtkernel.adoc:33]: Before creating manifests, we need to create intall-config.yaml using installer or prepare it manually first. To create it using installer:
$ ./openshift-install create install-config --dir=<installation_directory>

File: modules/installation-special-config-rtkernel.adoc [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+. Generate the Kubernetes manifests for the cluster:
++
+----
+$ ./openshift-install create manifests --dir=<installation_directory>
+----
+
+. Decide if you want to add the real-time kernel to worker or master nodes.
+
+. In the `openshift` directory, create a file (for example,
+`01-worker-realtime.yaml.yaml`) to define a MachineConfig
```
Comment (shellyyang1989) [inline modules/installation-special-config-rtkernel.adoc:43]: The file name 01-worker-realtime.yaml.yaml has 2 suffix. I suggest to change to 01-worker-realtime.yaml

File: modules/installation-special-config-rtkernel.adoc [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+object that applies a real-time kernel to the selected nodes (worker
+nodes in this case):
++
+----
+$ cat << EOF > 01-worker-realtime.yaml
```
Comment (shellyyang1989) [inline modules/installation-special-config-rtkernel.adoc:48]: I suggest to change the file name to 99-worker-realtime.yaml.

**Comment by shellyyang1989** [comment]
@sunilcio Could you please review the nodes change? Thanks

**Comment by sunilcio** [comment]
@shellyyang1989 sure, is the preview available at https://chris-grpa-1211--ocpdocs.netlify.app/openshift-enterprise/latest/installing/install_config/installing-customizing.html ?
@lyman9966 please help review this.

**Comment by ahardin-rh** [comment]
@sunilcio @shellyyang1989 @lyman9966 Thank you!
This content is for our webscale (Telco) distribution only. Preview builds are here:

http://file.rdu.redhat.com/~ahardin/April-30-2020/chris-GRPA-1211/installing/install_config/installing-customizing.html#installation-special-config-rtkernel_installing-customizing

http://file.rdu.redhat.com/~ahardin/April-30-2020/chris-GRPA-1211/nodes/nodes/nodes-nodes-working.html#nodes-nodes-rtkernel-arguments_nodes-nodes-working

**Comment by ahardin-rh** [comment]
Thank you for the reviews! I addressed your comments. If anything else is needed, we can address it in a separate PR.

**Comment by shellyyang1989** [comment]
@ahardin-rh Since real-time kernel is actually covered by edge team, I am inviting edge team guys to double check. @yprokule could you please provide your review? Installer and Nodes QE already provide some comments, please correct us if anything is improper. Thanks.

**Pull request merged**